_posts/2025-08-02-ordenar-score-asc-desc.md

```
@@ -0,0 +1,21 @@
+tags: [Cliente, Ejecutivo, Factura, Gestión comercial, Score]
+comments: false
+author: Catalina Pinilla
+---
+
+## Gestión comercial - Score.
+
+Ya que hay ocasiones que se necesita clasificar de una manera simple y rápida la información que buscas, por lo que se implementó la función de poder ordenar el score del valor mas alto al más bajo o al revés, para que puedas localizar el rango que necesitas de una manera mas eficiente.
+
+#### Aquí te mostramos como se visualiza esta nueva implementación.
+
+<video width="100%" controls>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: video

(MD033, no-inline-html)